dns/route53/client.go

```diff
@@ -70,3 +81,28 @@ func (c *Client) GetResourceRecordSetByName(zoneID, recordName, recordType strin
 
 	return resp.ResourceRecordSets[0], nil
 }
+
+// NOTE: depending on the given ChangeAction, a record can be created, updated or deleted
```

Comment:
Please refer to https://blog.golang.org/godoc and start the comment with the name of the thing you're documenting.

Reply:
Will be handled as if there would be no `NOTE:` and shows up in docs as expected. So, I'd like to leave it that way.

Reply:
This will fail `golint ./...`:

> dns/route53/client.go:85:1: comment on exported method Client.ChangeRecord should be of the form "ChangeRecord ..."

Reply:
Also see https://github.com/wireapp/srv-announcer/pull/11

Reply:
The purpose of a linter is not to add a line containing the necessary string to make it `exit 0`, but actually follow the guidelines it actually tries to verify are implemented.

> The convention is simple: to document a type, variable, constant, function, or even a package, write a regular comment directly preceding its declaration, with no intervening blank line […]